README.rst

```diff
@@ -261,6 +269,14 @@ To uninstall run
 
    python -m pip uninstall pyhf
 
+Documentation
+-------------
+
+For model scpecification, API reference, examples, and answers to FAQs visit the |pyhf documentation|_.
```

Comment:
```suggestion
For model specification, API reference, examples, and answers to FAQs visit the |pyhf documentation|_.
```

Reply:
Good catch. I'm a bit surprised that codespell allows that. :?